sdk/src/Core/Amazon.Util/AWSSDKUtils.cs

```diff
@@ -75,12 +77,6 @@ public static partial class AWSSDKUtils
         // Default value of progress update interval for streaming is 100KB.
         public const long DefaultProgressUpdateInterval = 102400;
 
-        internal static Dictionary<int, string> RFCEncodingSchemes = new Dictionary<int, string>
```

Comment:
I did check the entire SDK and couldn't find another reference so I got rid of it. Is it possible to entirely get rid of the different RFC implementations? If that is possible we could potentially even get rid of the custom code and replace it with the System.Net implementation? 

https://learn.microsoft.com/en-us/dotnet/api/system.net.webutility.urlencode?view=net-8.0